src/contracts/IbexaKernelTestTrait.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Test\Core;
+
+use Doctrine\DBAL\Connection;
+use Ibexa\Contracts\Core\Persistence\TransactionHandler;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\ContentTypeService;
+use Ibexa\Contracts\Core\Repository\LanguageService;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\Repository\ObjectStateService;
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\RoleService;
+use Ibexa\Contracts\Core\Repository\SearchService;
+use Ibexa\Contracts\Core\Repository\SectionService;
+use Ibexa\Contracts\Core\Repository\UserService;
+use Ibexa\Contracts\Core\Test\IbexaTestKernelInterface;
+use Ibexa\Contracts\Core\Test\Persistence\Fixture\FixtureImporter;
+use Ibexa\Core\Repository\Values\User\UserReference;
+use Ibexa\Tests\Core\Repository\LegacySchemaImporter;
+use RuntimeException;
+
+/**
+ *  @experimental
+ */
+trait IbexaKernelTestTrait
```

Comment:
As discussed, we cannot initialize that object in the `setUp` method, because `WebTestCase::createClient` boots the Kernel on it's own (and complains if a Kernel is already booted).

I'm okay with removing the trait, just keep in mind that for this to work now each and every instance where we call the `self::getServiceByClassName()` (or specific service declared in trait explicitly) will now need to be converted to `$this->getIbexaCore()->getServiceByClassName()`.

Fixed in 2c71681a.